README.md

```diff
@@ -17,16 +17,35 @@ RKE2 is a fully compliant Kubernetes distribution, with the following changes:
   - Nginx Ingress Controller
   - Metrics Server
 
-## Quick Start
+## Installation
+
+RKE2 comes with a convenient and configurable installation script that allows for system configuration, binary release download, and init script setup. 
+
+### Quick Start
 
 The `install.sh` script provides a convenient way for installing rke2 server and agent, to install rke2 service:
 
 ```sh
-INSTALL_RKE2_VERSION=v0.0.1-alpha.4 ./install.sh
+INSTALL_RKE2_VERSION=v0.0.1-alpha.7 ./install.sh
+```
+
+### CIS Mode
+
+If you plan to run your RKE2 cluster in CIS mode, the installation script offers the ability to install RKE2 in preparation for this. Use the `INSTALL_RKE2_CIS_MODE` environment variable. This will configure the system for RKE2 to run in CIS compliant mode. This includes the creation of a user "etcd", updates a number of kernel parameters, and installs a systemd or openrc unit file that passes the CIS mode flag to the binary (`--profile=cis-1.5`). Please reference the installation script (`install.sh`) for CIS specifics.
```

Comment:
nit: might clarify that it doesn't just create the etcd user, but runs etcd as this user (instead of as root?) as well.

Reply:
This is describing what the install script does. It doesn't run etcd but just creates the user. RKE2 configures the manifest and runs the pods as the etcd user. I'm putting additional docs together for the rke2 binary run in cis mode that will be added later.